backend/webui_service/webui_init.go

```diff
@@ -149,7 +149,7 @@ func (webui *WEBUI) Start() {
 	self := webui_context.WEBUI_Self()
 	self.UpdateNfProfiles()
 
-	router.NoRoute(ReturnPublic())
+	// router.NoRoute(ReturnPublic())
```

Comment:
This line can not be commented, cause this route is used to browse the static files (e.g. css, js, html).

I think the better way is to check the `context.Request.RequestURI` in the `ReturnPublic()`.
If It's unsafe , should call `context.JSON(404, gin.H{"code": "PAGE_NOT_FOUND", "message": "Page not found"})`.